encode.go

```diff
@@ -154,6 +154,9 @@ func (e *Encoder) Encode(v interface{}) error {
 				return e.encObject(p)
 			}
 
+			if _, ok := checkPOJORegistry(vv.Type().String()); ok {
```

Comment:
just call `encObject` and not need to check, which will be done in func `encObject`